include/orc/orc.hpp

```diff
@@ -46,7 +46,13 @@ bool filter_report(const odrv_report& report);
 
 /**************************************************************************************************/
 
-std::vector<odrv_report> orc_process(const std::vector<std::filesystem::path>&);
+std::vector<odrv_report> orc_process(std::vector<std::filesystem::path>&&);
+
+namespace orc {
```

Comment:
I've started putting ORC calls into an `orc` namespace. We really need to address this more completely as part of issue #20.